app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SessionsController < ApplicationController
+  skip_before_action :authenticate_user, except: :destroy
+  before_action :redirect_if_authenticated, except: :destroy
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.find_by(email: params[:user][:email])
```

Comment:
you can use '&'  to handle cases where params[:user][:email] is nil. This way, it won't raise an error if the email parameter is missing.